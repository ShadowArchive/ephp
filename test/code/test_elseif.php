<?php

$data = true;
if ($data == true) {
    print "OK\n";
} elseif ($data != true) {
    print "NO OK\n";
}

switch ($data) {
    case true: print "OK\n";
    case 1: print "and OK\n";
    default: print "and other OK\n";
}

