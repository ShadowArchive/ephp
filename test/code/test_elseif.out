OK
OK
and OK
and other OK
